packages/aws-cdk-lib/aws-s3/test/bucket.test.ts

```diff
@@ -3134,6 +3134,30 @@ describe('bucket', () => {
     });
   });
 
+  test('Inventory Ids are shortened to 64 characters', () => {
+    // Given
+    const stack = new cdk.Stack();
+
+    const inventoryBucket = new s3.Bucket(stack, 'InventoryBucket');
+    new s3.Bucket(stack, 'AVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVery&LongNodeIdName', {
```

Comment:
can you add a test or change some characters in this one to show the unallowed characters being removed?

Reply:
There is an & in the string. Does this suffice, or should I make it more obvious?

Reply:
Sorry, I missed that. I think a few more would make it easier to see. Thanks!